src/style.css

```
@@ -1,3 +1,74 @@
+:root {
+  --blue-color: #00acdc;
+}
+
+html,
+body {
+  margin: 0;
+  padding: 0;
+}
+
 body {
+  font-family: Roboto, sans-serif;
+  text-transform: uppercase;
+}
+
+header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+nav {
+  height: 60px;
+}
+
+nav ul {
+  list-style-type: none;
   margin: 0;
+  padding: 0;
+  display: flex;
+}
```

use class selectors fro styling. check cases below as well
 `nav` styling is redundant
```suggestion
.header {
  display: flex;
  align-items: center;
  justify-content: space-between;
  padding: 0 50px;
}

.nav__list {
  list-style-type: none;
  margin: 0;
  padding: 0;
  display: flex;
}
```